frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -71,32 +77,78 @@ const MapSideBarContainer: React.FunctionComponent<IMapSideBarContainerProps> =
     (activePropertyDetail?.parcelDetail as any)?.rowVersion,
   ]);
 
+  const renderParcelDetailsContainer = () => {
+    return parcelId !== cachedParcelDetail?.id ? (
+      <>
+        <Spinner animation="border"></Spinner>
+      </>
+    ) : (
+      <ParcelDetailContainer
+        persistCallback={(data: IParcel) => {
+          if (!showSideBar) {
+            setShowSideBar(true);
+          }
+          overrideParcelId(data?.id === '' ? undefined : data?.id);
+        }}
+        onDelete={() => {
+          dispatch(parcelsActions.storeParcelDetail(null));
+          refreshParcels();
+          setShowSideBar(false);
+        }}
+        parcelDetail={cachedParcelDetail}
+        disabled={disabled}
+        loadDraft={loadDraft}
+        properties={properties}
+        mapClickMouseEvent={leafletMouseEvent}
+      />
+    );
+  };
+
+  const getSidebarTitle = (): React.ReactNode => {
+    switch (context) {
+      case SidebarContextType.ADD_BUILDING:
+        return (
+          <>
+            <BuildingSvg className="svg" /> Submit Building (to inventory)
+          </>
+        );
+      case SidebarContextType.ADD_RAW_LAND:
+        return (
+          <>
+            <LandSvg className="svg" /> Submit Raw Land (to inventory)
+          </>
+        );
+      case SidebarContextType.VIEW_BUILDING:
+        return 'Building Details';
+      case SidebarContextType.VIEW_RAW_LAND:
+        return 'Raw Land Details';
+      default:
+        return 'Submit a Property';
+    }
+  };
+
+  const render = (): React.ReactNode => {
+    switch (context) {
+      case SidebarContextType.ADD_BUILDING:
+        return <BuildingForm />;
+      case SidebarContextType.ADD_RAW_LAND:
+        return <BuildingForm />;
```

Comment:
I'll change this on my end when implementing the raw land form

Reply:
Yes, sir. Just a placeholder